rust/metaplex/program/src/instruction.rs

```diff
@@ -389,24 +392,25 @@ pub enum MetaplexInstruction {
     ///   5. `[writable]` Vault account
     ///   6. `[writable]` Fraction mint of the vault
     ///   7. `[]` Auction
-    ///   8. `[]` Your BidderMetadata account
-    ///   9. `[]` Your Bidder account - Only needs to be signer if payer does not own
-    ///   10. `[signer]` Payer
-    ///   11. `[]` Token program
-    ///   12. `[]` Token Vault program
-    ///   13. `[]` Token metadata program
-    ///   14. `[]` Store
-    ///   15. `[]` System
-    ///   16. `[]` Rent sysvar
-    ///   17. `[writable]` Prize tracking ticket (pda of ['metaplex', program id, auction manager key, metadata mint id])
-    ///   18. `[writable]` New Metadata key (pda of ['metadata', program id, mint id])
-    ///   19. `[writable]` New Edition (pda of ['metadata', program id, mint id, 'edition'])
-    ///   20. `[writable]` Master Edition of token in vault V2 (pda of ['metadata', program id, master metadata mint id, 'edition']) PDA is relative to token metadata.
-    ///   21. `[writable]` Mint of new token
-    ///   22. `[writable]` Edition pda to mark creation - will be checked for pre-existence. (pda of ['metadata', program id, master metadata mint id, 'edition', edition_number])
+    ///   8. `[]` Auction extended
```

Comment:
Yah again, can't do this - you'll break older UIs. Got to put it at the end, or make a new end point.